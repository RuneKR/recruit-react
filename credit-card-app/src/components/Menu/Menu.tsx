import React, { Component } from "react";
import { Menu as AntdMenu } from "antd";

class Menu extends Component {
  render() {
    return (
      <AntdMenu>
        <AntdMenu.Item key="home">
          <a href="">Home</a>
        </AntdMenu.Item>
        <AntdMenu.Item key="contactus">
          <a href="">Contact Us</a>
        </AntdMenu.Item>
      </AntdMenu>
    );
  }
}
export default Menu;
