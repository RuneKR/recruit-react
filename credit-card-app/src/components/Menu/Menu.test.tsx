import React from "react";
import renderer from "react-test-renderer";
import Menu from "./Menu";

describe("Menu Component", () => {
  test("menu rendered correctly", () => {
    const component = renderer.create(<Menu />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
