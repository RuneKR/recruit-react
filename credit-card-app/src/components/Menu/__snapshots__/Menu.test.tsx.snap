// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Component menu rendered correctly 1`] = `
<ul
  className="ant-menu ant-menu-light ant-menu-root ant-menu-vertical"
  onMouseEnter={[Function]}
  onTransitionEnd={[Function]}
  role="menu"
  style={Object {}}
>
  <li
    className="ant-menu-item"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    role="menuitem"
    style={Object {}}
  >
    <a
      href=""
    >
      Home
    </a>
  </li>
  <li
    className="ant-menu-item"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    role="menuitem"
    style={Object {}}
  >
    <a
      href=""
    >
      Contact Us
    </a>
  </li>
</ul>
`;

exports[`Menu rendered correctly 1`] = `
<ul
  className="ant-menu ant-menu-light ant-menu-root ant-menu-vertical"
  onMouseEnter={[Function]}
  onTransitionEnd={[Function]}
  role="menu"
  style={Object {}}
>
  <li
    className="ant-menu-item"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    role="menuitem"
    style={Object {}}
  >
    <a
      href=""
    >
      Home
    </a>
  </li>
  <li
    className="ant-menu-item"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    role="menuitem"
    style={Object {}}
  >
    <a
      href=""
    >
      Contact Us
    </a>
  </li>
</ul>
`;
